avalon/tools/workfiles/app.py

```diff
@@ -166,10 +212,18 @@ def refresh(self):
             template = self.get_work_file(template)
             template = "^" + template + "$"           # match beginning to end
 
+            # Match with ignore case on Windows due to the Windows
+            # OS not being case-sensitive. This avoids later running
+            # into the error that the file did exist if it existed
+            # with a different upper/lower-case.
+            kwargs = {}
+            if platform.system() == "Windows":
+                kwargs["flags"] = re.IGNORECASE
```

Comment:
This is a fix for a bug I found when there were matching files with the template however they had a different case (uppercase vs lowercase) - previously it would ignore the match and it could reach the "This a bug, file exists.." assertion below. That is patched by (on Windows) matching without case sensitivity because `os.path.exists` would detect is as existing.